cpp/daal/src/algorithms/objective_function/logistic_loss/logistic_loss_dense_default_batch_impl.i

```diff
@@ -271,39 +270,130 @@ services::Status LogLossKernel<algorithmFPType, method, cpu>::doCompute(const al
             fPtr  = fScalable.get();
             sgPtr = sgScalable.get();
         }
-        //f = X*b + b0
-        applyBetaThreaded(x, b, fPtr, n, p, parameter->interceptFlag);
 
+        const bool bL1 = parameter->penaltyL1 > 0;
+        const bool bL2 = parameter->penaltyL2 > 0;
+
+        const size_t iFirstBeta   = parameter->interceptFlag ? 0 : 1;
+        const algorithmFPType div = static_cast<algorithmFPType>(1) / static_cast<algorithmFPType>(n);
+
+        const size_t nRowsInBlock = 512;
+        const size_t nDataBlocks  = n / nRowsInBlock + !!(n % nRowsInBlock);
+
+        TlsMem<algorithmFPType, cpu> tlsData(2 * nRowsInBlock);
+
+        TArrayScalable<algorithmFPType, cpu> values;
+        if (valueNT)
         {
-            DAAL_ITTNOTIFY_SCOPED_TASK(sigmoids);
```

Comment:
seems deleted DAAL_ITTNOTIFY_SCOPED_TASK profiling macro should be reverted in whole file.